test/ModuleInterface/features.swift

```diff
@@ -189,3 +189,33 @@ public func unavailableFromAsyncFunc() { }
 public func noAsyncFunc() { }
 
 // CHECK-NOT: extension FeatureTest.MyActor : Swift.Sendable
+
+// CHECK: #if compiler(>=5.3) && $LexicalLifetimes
+// CHECK-NEXT: @_noEagerMove public struct Permanent {
+// CHECK-NEXT: }
+// CHECK-NEXT: #else
+// CHECK-NEXT: public struct Permanent {
+// CHECK-NEXT: }
+// CHECK-NEXT: #endif
+@_noEagerMove
+public struct Permanent {}
```

Comment:
I have a minor preference for new tests for printing of kinds of attributes/declarations go in separate test cases under `ModuleInterfaces/`. At some point I'm probably going to clean up the old feature guards we don't need to print anymore and blow up this particular test case because it tests a lot of unrelated language features and probably shouldn't be one monolithic test.

Reply:
Moved to `test/ModuleInterface/feature-LexicalLifetimes.swift`.